src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java

```diff
@@ -496,7 +496,13 @@ private void process(@NotNull String entryPath, @NotNull Archive archive, @Nulla
 
     @Override
     protected void onFile(@NotNull String entryPath, @NotNull Archive archive, @NotNull Entry entry) throws IOException, ConverterException {
-        process(entryPath, archive, entry);
+        try {
+            process(entryPath, archive, entry);
+        }catch (ConverterException ex){
+            throw new ConverterException("ConverterException occured on path " + entryPath + " with message: " + ex.getMessage(), ex);
+        }catch(IOException ex){
```

Comment:
same here: please stick to the formatting the project which is whitespace between } and catch and between ) and {